python/data/chap3/README.rst

```diff
@@ -0,0 +1,28 @@
+Chapter 3 - Logical Record Syntax
+=================================
+
+Testfiles for the general layout of (Explicitly) Formatted Logical Records,
+defined in Chapter 3 of rp66v1. Each file is an EFLR component, with exception
+of sul.dlis.part and start.dlis.part.
+
+============================ =================================================
+File / directory             Description     
+============================ =================================================
+sul.dlis.part                Default Storage Unit Label
+start.dlis.part              SUL + a default set definition
```

Comment:
1) then + all the VR and LRSH bytes

2)  if you want:
Bytes 50 and 51 contain VR length. Bytes 54 and 55 contain LRSH length. Bytes are supposed to be overwritten by test framework